src/NuGetForUnity/Editor/NugetConfigFile.cs

```diff
@@ -147,6 +161,11 @@ public void Save(string filePath)
             addElement.Add(new XAttribute("value", savedRepositoryPath));
             config.Add(addElement);
 
+            addElement = new XElement("add");
+            addElement.Add(new XAttribute("key", "PackagesConfigPath"));
```

Comment:
I would move `"PackagesConfigPath"` to a constant so we have no magic value, I did it for all new settings I added

Reply:
Fixed. I could do the same for the rest of the keys, I agree it would seem more clean that way.